src/meshes/p4est_mesh.jl

```diff
@@ -1427,13 +1441,14 @@ function balance!(mesh::P4estMesh{3}, init_fn=C_NULL)
   p8est_balance(mesh.p4est, P8EST_CONNECT_FACE, init_fn)
 end
 
+p4est_partition_allow_for_coarsening!(mesh::P4estMesh, p4est_partition_allow_for_coarsening) = mesh.p4est_partition_allow_for_coarsening = p4est_partition_allow_for_coarsening
```

Comment:
```suggestion
"""
    p4est_partition_allow_for_coarsening!(mesh::P4estMesh, new_value::Bool)

Set the parameter `p4est_partition_allow_for_coarsening` of the `mesh` to the `new_value`.
"""
p4est_partition_allow_for_coarsening!(mesh::P4estMesh, p4est_partition_allow_for_coarsening) = mesh.p4est_partition_allow_for_coarsening = p4est_partition_allow_for_coarsening
```

Reply:
Should we also call the parameter `new_value` then?

Reply:
I don't have a strong opinion on this - sounds reasonable though

Reply:
I added the docstring and renamed the parameter.